nipoppy_cli/nipoppy/utils.py

```diff
@@ -36,87 +36,97 @@
 
 # descriptions for common fields in the Pydantic models
 FIELD_DESCRIPTION_MAP = {
-    "bids_id": "BIDS-compliant participant identifier (e.g., sub-01)",
-    "participant_id": "Participant identifier",
-    "session": "BIDS-compliant identifier imaging session (e.g., ses-1)",
-    "visit": "Visit identifier",
+    "participant_id": "Participant identifier, without the BIDS prefix",
+    "session_id": "Imaging session identifier, without the BIDS prefix",
+    "bids_participant": "Participant identifier with BIDS prefix (e.g., sub-01)",
+    # only for bagel
+    "bids_session": "Participant identifier with BIDS prefix (e.g., sub-01)",
```

Comment:
we define `bids_session` only for bagel but not for other places? Elsewhere we are using `session` ?
Also a typo in the example.

Reply:
Removed the comment as it was not entirely accurate -- initially I kept the entry as `session` but I think I changed it in the `FIELD_DESCRIPTION_MAP` (so now it doesn't match the bagel column, but will make things easier when we eventually drop `session` from the bagel)